src/test/unit/pid_unittest.cc

```diff
@@ -84,6 +88,10 @@ extern "C" {
     void systemBeep(bool) { }
     bool gyroOverflowDetected(void) { return false; }
     float getRcDeflection(int axis) { return simulatedRcDeflection[axis]; }
+    float getRcCommandDelta(int axis) { return simulatedRcCommandDelta[axis]; }
```

Comment:
Please also add

    float getRcDeflectionRaw(int axis) { return simulatedRcDeflection[axis]; }
    
But still test is failing with that but more graceful.